.github/workflows/interface-unit-tests.yml

```
@@ -534,7 +534,7 @@ jobs:
       pytest_xml_file_path: '${{ inputs.job_name_prefix }}external-libraries-tests (${{ matrix.python-version }})${{ inputs.job_name_suffix }}.xml'
       additional_pip_packages: |
         pyzx matplotlib stim quimb==1.11.0 mitiq ply optax scipy-openblas32>=0.3.26 qualtran openqasm3 antlr4_python3_runtime xdsl==0.45 filecheck
-        jax==0.6.0 jaxlib==0.6.0
+        jax==0.6.2 jaxlib==0.6.2
```

We should be getting the jax version from the `interface-dependency-versions.yml` workflow, same as the TF version. That workflow currently specifies 0.6.2 as the JAX version.

```suggestion
        ${{ needs.default-dependency-versions.outputs.jax-version }}
```